package.json

```diff
@@ -763,6 +764,7 @@
     "prettier": "^2.2.0",
     "pretty-ms": "5.0.0",
     "proxyquire": "1.8.0",
+    "q": "^1.5.1",
```

Comment:
All dependencies used by our code should be listed in our `package.json` file as either a dependency or a dev-dependency. The `q` package is being used by `x-pack/plugins/security_solution/scripts/beat_docs/build.js` which expected `q` to be present. However it was only present as a transient dependency of another package that this PR removed.

I've added `q` to our `devDependencies` list because I think `x-pack/plugins/security_solution/scripts/beat_docs/build.js` isn't being used in production. @elastic/security-solution  could you please verify this?